javalib/src/main/scala/java/util/concurrent/ConcurrentLinkedQueue.scala

```diff
@@ -0,0 +1,153 @@
+package java.util.concurrent
+
+import java.util._
+import scala.collection.JavaConversions._
+
+class ConcurrentLinkedQueue[E]()
+    extends AbstractQueue[E] with Queue[E] with Serializable {
+
+  def this(c: Collection[_ <: E]) = {
+    this()
+    addAll(c)
+  }
+
+  import ConcurrentLinkedQueue._
+
+  private var head: Node[E] = null
+  private var last: Node[E] = null
+
+  private var _size: Double = 0
+
+  override def add(e: E): Boolean =
+    if (e == null) throw new NullPointerException()
+    else {
```

Comment:
According to https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#ifelse, the then and else parts should have the same "look", hence:
```scala
if (e == null) {
  throw new NullPointerException()
} else {
```
Also, according to https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#braces, The entire body of `add()` should have `{}`.